os_migrate/roles/prelude_common/templates/clouds.yaml.j2

```diff
@@ -0,0 +1,21 @@
+clouds:
+  src:
+  {% if os_migrate_src_auth is defined and os_migrate_src_auth|length %}
+    auth: {{os_migrate_src_auth|to_nice_yaml(indent=2)|indent(2)}}
```

Comment:
I think this one will have to be on 2 lines to format it like a proper YAML dict, somehting like:

```
auth:
  {{os_migrate_src_auth|to_nice_yaml(indent=2)|indent(2)}}
```

But the indentation numbers are probably off still.